i18n/pl.toml

```diff
@@ -0,0 +1,191 @@
+other = "Cleaning up"
+
+[community_events_calendar]
+other = "Kalendarz wydarzeń"
+
+[community_forum_name]
+other = "Forum"
+
+[community_github_name]
+other = "GitHub"
+
+[community_slack_name]
+other = "Slack"
+
+[community_stack_overflow_name]
+other = "Stack Overflow"
+
+[community_twitter_name]
+other = "Twitter"
+
+[community_youtube_name]
+other = "YouTube"
+
+[deprecation_warning]
+other = " dokumentacja nie jest juz aktualizowana. Wyświetlona jest wersja archiwalna. Po aktualną dokumentację zajrzyj na"
+
+[deprecation_file_warning]
+other = "Przestarzały"
+
+[docs_label_browse]
+other = "Przeglądaj dokumentację"
+
+[docs_label_contributors]
+other = "Współautorzy"
+
+[docs_label_i_am]
+other = "Jestem..."
+
+[docs_label_users]
+other = "Użytkownicy"
+
+[feedback_heading]
+other = "Twoja opinia"
+
+[feedback_no]
+other = "Nie"
+
+[feedback_question]
+other = "Czy ta strona była przydatna?"
+
+[feedback_yes]
+other = "Tak"
+
+[latest_version]
+other = "to najnowsza wersja."
+
+[layouts_blog_pager_prev]
+other = "<< Poprzedni"
+
+[layouts_blog_pager_next]
+other = "Następny >>"
+
+[layouts_case_studies_list_tell]
+other = "Opowiedz swoją historię"
+
+[layouts_docs_glossary_aka]
+other = "Znany tez jako"
+
+[layouts_docs_glossary_description]
+other = "Celem tego słownika jest przedstawienie wszechstronnej, ujednoliconej listy terminologii związanej z Kubernetes. Słownik zawiera terminy specyficzne dla Kubernetes, a także pojęcia bardziej ogólne, dające lepsze zrozumienie kontekstu."
+
+[layouts_docs_glossary_deselect_all]
+other = "Odznacz wszystko"
+
+[layouts_docs_glossary_click_details_after]
+other = "po dokładniejsze wytłumaczenie."
+
+[layouts_docs_glossary_click_details_before]
+other = "Kliknij w"
+
+[layouts_docs_glossary_filter]
+other = "Znajdź pojęcia według etykiet"
+
+[layouts_docs_glossary_select_all]
+other = "Zaznacz wszystko"
+
+[layouts_docs_partials_feedback_improvement]
+other = "zaproponować poprawkę"
+
+[layouts_docs_partials_feedback_issue]
+other = "Otwórz zgłoszenie w repozytorium GitHub, jeśli chcesz "
+
+[layouts_docs_partials_feedback_or]
+other = "lub"
+
+[layouts_docs_partials_feedback_problem]
+other = "zgłosić problem"
+
+[layouts_docs_partials_feedback_thanks]
+other = "Dziękujemy za informację zwrotną. Jeśli masz konkretne pytanie dotyczące użycia Kubernetes, odwiedź"
+
+[layouts_docs_search_fetching]
+other = "Pobieram wyniki.."
+
+[main_by]
+other = "przez"
+
+[main_cncf_project]
+other = """Nasz projekt jest uznany przez <a href="https://cncf.io/">CNCF</a> za dojrzały</p>"""
+
+[main_community_explore]
+other = "Poznaj społeczność"
+
+[main_contribute]
+other = "Wnieś swój wkład"
+
+[main_copyright_notice]
+other = """The Linux Foundation &reg;. All rights reserved. The Linux Foundation has registered trademarks and uses trademarks. For a list of trademarks of The Linux Foundation, please see our <a href="https://www.linuxfoundation.org/trademark-usage" class="light-text">Trademark Usage page</a>"""
```

Comment:
nit: not yet localized

Reply:
I'm not going to translate this copyright notice.

Reply:
There are certainly localizations that skip translating it. For a counter-example, see https://kubernetes.io/zh/